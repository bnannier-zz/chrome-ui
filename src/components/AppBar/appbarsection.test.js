import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import { AppBar, AppBarSection } from "../";

describe("AppBarSection Tests", () => {
  it("Check Sections Left and Right", function() {
    const wrapper = shallow(
      <div>
        <AppBar>
          <AppBarSection left>Left Side</AppBarSection>
          <AppBarSection right>Right Side</AppBarSection>
        </AppBar>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
