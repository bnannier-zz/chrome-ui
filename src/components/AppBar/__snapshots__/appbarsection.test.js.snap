// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBarSection Tests Check Sections Left and Right 1`] = `
<div>
  <AppBar>
    <AppBarSection
      left={true}
    >
      Left Side
    </AppBarSection>
    <AppBarSection
      right={true}
    >
      Right Side
    </AppBarSection>
  </AppBar>
</div>
`;
